gradle/testing/randomization.gradle

```diff
@@ -85,7 +85,6 @@ allprojects {
           [propName: 'tests.timezone', value: "random", description: "Sets the default time zone tests should run with."],
           // filtering
           [propName: 'tests.filter', value: null, description: "Applies a test filter (see :helpTests)."],
-          [propName: 'tests.slow', value: true, description: "Enables or disables @Slow tests."],
```

Comment:
added https://github.com/apache/solr/pull/951/commits/f87fe8ad969e41f8957db3821b1d1593d25f4720 commit, hope that's okay.